apps/email/js/sync.js

```diff
@@ -501,8 +512,8 @@ define(function(require) {
 
           sendNotification(
             BACKGROUND_SEND_NOTIFICATION_ID,
-            mozL10n.get('background-send-error-title'),
-            data.localizedDescription,
+            { raw: 'background-send-error-title' },
+            { raw: data.localizedDescription },
```

Comment:
here you take `{raw: data.localizedDescription}`. 50 lines earlier the code was using `mozL10n.get` to localize the `descId`. You should not use it as `raw`, but intead pass `descId` which is an L10nID.